pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/EndOfStreamAction.java

```diff
@@ -16,6 +16,11 @@
 
 package org.apache.pulsar.reactive.client.api;
 
+/**
+ * Action to perform once the end of the stream is reached.
+ *
+ * @author Lari Hotari
```

Comment:
IIRC generally, we don't encourage using the `@author` tag. Is there any reason for doing this?

Reply:
It's in the checkstyle rules (derivative of Spring rules)
If we want to remove it, we should do it in another PR.

Reply:
I filed #67 to remove author tags. I think it's better to get rid of them.

Reply:
Let's get the in-process PRs merged and then handle that.